README.md

```diff
@@ -14,21 +14,21 @@ You can [create an issue to contact XboxDev maintainers](https://github.com/Xbox
 
 We currently provide the following services:
 
-* Discord for communication
-* XboxDevWiki to document research
-* GitHub organization to host tools and files
+* [Discord](https://discord.gg/XUDsBYD) for communication
+* [XboxDevWiki](http://xboxdevwiki.net) to document research
+* [GitHub organization](https://github.com/XboxDev) to host tools and files
 
 
 ## Want to help? Contact us!
 
 Please come chat with us:
 
-* XboxDev on Discord
+* [XboxDev on Discord](https://discord.gg/XUDsBYD)
 
 and for closely related projects:
 
-* #xqemu on Freenode IRC (Webchat)
-* Cxbx-Reloaded on gitter.im
+* [#xqemu on Freenode IRC](irc://chat.freenode.net/xqemu) ([Webchat](https://webchat.freenode.net/?channels=%23xqemu))
+* [Cxbx-Reloaded on gitter.im](https://gitter.im/Cxbx-Reloaded/Lobby)
```

Comment:
Probably the "Original Xbox Emulation" Discord is a better way to reach the CXBX-R guys. @PatrickvL and @LukeUsher can confirm

Reply:
The contact information was copied from XboxDevWiki and is from early 2017.
The Cxbx-Reloaded README does indeed list both now (only Discord back then).

From experience I can say that the gitter is dead. I also recently asked and was told they primarily use Discord.
I wonder if we should refomat the entire paragraph slightly to something like:

> - [XQEMU](xqemu-web-or-project): [#xqemu on Freenode IRC](link) ([Webchat](link))
> - [Cxbx-Reloaded](cxbx-r-web-or-project): [Discord](link) or [Gitter.im](link)

Reply:
I'm of two minds weather to even link the Gitter at all at this point: Everyone just gravitated towards discord.